modules/lite/src/deckgl.js

```diff
@@ -12,13 +12,13 @@ const CANVAS_STYLE = {
   height: '100%'
 };
 
-// Add missing props for compatibility
-function normalizeProps(props) {
-  // Support old "geospatial view state as separate props" style
-  if (!props.initialViewState) {
-    const {longitude, latitude, zoom, pitch = 0, bearing = 0} = props;
-    props.initialViewState = props.viewState || {longitude, latitude, zoom, pitch, bearing};
+// Supports old "geospatial view state as separate props" style
+function getViewState(props) {
```

Comment:
This is taken from `react/deckgl.js`. Do we want to support it on the core level?

Reply:
> This is taken from react/deckgl.js. Do we want to support it on the core level?

Per my comment below I intentionally removed it from core, and deprecated it in React. 

I felt while the top level props are a little simpler for new (geospatial) users, they do not extend/mesh well with the viewState system and overall not worth supporting as a special case.

That decision can of course be reversed. Especially if we are supporting them everywhere but in core, that does not make much sense either.